samples/gocdk-blob/blob.ct

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 The Go Cloud Development Kit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Tests of the gocdk-blob program.
+
+# Create a subdirectory so we can use fileblob.
+$ mkdir bucket
+
+# List an empty bucket: valid, but no output.
+$ gocdk-blob ls file://${ROOTDIR}/bucket
+
+$ echof hello.txt hello world
+$ gocdk-blob upload file://${ROOTDIR}/bucket hw < hello.txt
+
+$ gocdk-blob ls file://${ROOTDIR}/bucket
+hw
+
+$ gocdk-blob download file://${ROOTDIR}/bucket hw
+hello world
+
+# Error downloading a bucket that doesn't exist.
+$ gocdk-blob download file://${ROOTDIR}/bucket noexist --> FAIL
+gocdk-blob: Failed to read "noexist": blob (code=NotFound): stat ${ROOTDIR}/bucket/noexist: no such file or directory
```

Comment:
The exact error string is not going to be portable.

Reply:
This is a hard problem. We could easily come up with some sort of approximate match (e.g. look only at the prefix of the line; or allow regexps). But then how would updating work? 

I'm going to omit this test case pending a decent solution.

Reply:
SGTM. Can you file an issue?